src/main/java/org/apache/commons/collections4/CollectionUtils.java

```diff
@@ -609,6 +609,33 @@ public static boolean isEqualCollection(final Collection<?> a, final Collection<
         return isEqualCollection(collect(a, transformer), collect(b, transformer));
     }
 
+    /**
+     * Returns the hash code of Collection through the hash method of Equator.
```

Comment:
Maybe match the text in the `@param` docs: "Returns the hash code the the input collection using the hash method of the equator."?